lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1260,18 +1262,34 @@ def create_ReAuthRequest(self, imsi, policy_id, flow_list, qos):
         qos = QoSInformation(qci=qos["qci"])
 
         # Get sessionid
-        req = GetDirectoryFieldRequest(id=imsi, field_key="session_id")
-        try:
-            res = self._directorydstub.GetDirectoryField(
-                req, DEFAULT_GRPC_TIMEOUT
-            )
-        except grpc.RpcError as err:
-            logging.error(
-                "GetDirectoryFieldRequest error for id: %s! [%s] %s",
-                imsi,
-                err.code(),
-                err.details(),
-            )
+        #TODO: remove retries
+        i = 0
+        MAX = 3
+        while i < MAX:
+            req = GetDirectoryFieldRequest(id=imsi, field_key="session_id")
+            try:
+                res = self._directorydstub.GetDirectoryField(
+                    req, DEFAULT_GRPC_TIMEOUT
+                )
+            except grpc.RpcError as err:
+                print(
+                    "error: GetDirectoryFieldRequest error for id: %s! [%s] %s",
+                    imsi,
+                    err.code(),
+                    err.details(),
+                )
+            if req != None:
+                i = MAX
```

Comment:
We don't need to check this. If you really want to it is better done just after line 1269 and use break. Also should not this be `req == None`?

Reply:
this is just temporal
This is only failing on CI and right now we were not getting any message. 
Once i get the information it will be gone

Reply:
Hey @ulaskozat  this code is temporal. Right now we are not getting any message when it fails. This is only failing on CI. So i want to get all the content possible

The logic with this retries is: if there is a request we exit the loop. If there is no request we wait and increas `i`